### ISSUE TITLE
model zoo 

### ISSUE THREAD

**Issue by Chenyating** [open-mmlab/mmsegmentation#62]
Your repo is great.

I found that in Model ZOO, you gave models of different Lr SCHD（20000， 40000， 80000 and 160000). I have some problem.
1. Why do you give models of different of Lr SCHD instead of the best one?

2. If I want to use the models as pretrained model , which model should I choose?

3. Why did you use steps to design LR SCHD instead of epochs?

**Comment by xvjiarui** [comment]
Hi @Chenyating 
1. The models dubbed with 40k iteration are not picked from training process for 80k iteration. In 40k and 80k schedules, the learning rate decays differently (0.01 -> 0 in 40k iters vs 0.01 -> 0 in 80k iters). Models with different schedules are provided for the illustration on the impact of LR schedule. 
2. Different models have different properties, you may start with PSPNet or DeepLabV3+. 
3. Iteration based training is more flexible than epoch based, which is also the practice in the field.

**Comment by Chenyating** [comment]
Thank you for your response.
 I have a few more questions I'd like you to answer。
1. I do not understand why iteration based training is more flexible than epoch based, can you describe it in more detail.
2. If I want to train Deeplabv3 + on my own dataset and use the model you trained on Cityscapes as a pretrained model, do I choose the best model or choose the model based on the Lr SCHD I set.
3. I also found you give models of different crop size (769*768 and 512* 1024) on cityscapes datasets. Do you just want to show the effect of different crop size， or are there other reasons？

**Comment by xvjiarui** [comment]
Hi @Chenyating 
1. Iteration based allows us to evaluate whenever we like. If we are using epoch based, we won't be able to validate in the middle of the epoch. Given that some dataset is really large, it may not be ideal. Iteration based training is consistent with our LR decay policy. 
2. You may choose the one with higher performance. 
3. 769x769 and 512x1024 are two common practice in the field, on the one head, we do want to demonstrate the effect of different crop size, on the other hand, we would like to provide more settings for fair comparison. :)
